backend/services/dynamodb_service.py

```
@@ -130,6 +129,23 @@ def create_iam_finding(self, finding_data: Dict[str, Any]) -> bool:
+        kwargs: Dict[str, Any] = {}
+        try:
+            while True:
+                response = self.iam_findings.scan(**kwargs)
+                items.extend(response.get("Items", []))
+                lek = response.get("LastEvaluatedKey")
+                if not lek:
+                    break
+                kwargs["ExclusiveStartKey"] = lek
+            return items
+        except ClientError as e:
+            logger.error(f"DynamoDB error scanning IAM findings: {str(e)}")
```

Ruff (0.15.9)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)